int64                   id
geometry_msgs/Point     position
float64[]               covariance
float32                 weight
geometry_msgs/Twist     velocity
string[]                required_sensor